test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/DotnetCliUtil.cs

```diff
@@ -105,6 +108,12 @@ public static void VerifyNoClear(string path)
         /// </returns>
         public static string GetXplatDll()
         {
+            var xplatDll = Path.Combine(BuildOutputDirectory, XPlatDll);
```

Comment:
Discussed it offline with @alpaix. Suggested to check if this breaks linux and mac build process. 

Otherwise LGTM!